editor-extension-collection/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,6 @@
+[[extensions]]
```

Comment:
I wonder if we should include a second collection in here commented out as well, to demonstrate that they can copy/paste to add more, instead of using the CLI again?

(We should obviously verify this works as expected too.)

Reply:
hmm nested arrays in TOML can get messy, but adding another `[[extensions]]` element to the same file should be OK. The CLI _should_ just pick it up as an additional extension.

We can see how this looks when implemented though!

Reply:
Sounds good, just to clarify so for now will you like me to add a comment for a second extension or we can leave it out?

Reply:
Why don't we put one in as a reminder to test it out, it can always be removed later.

Reply:
Updated the template! Feel free to check it out here https://github.com/Shopify/extensions-templates/pull/93/commits/7ce5534a71299c0140c937a14d0ebe197a7d7467. Also updated the locale so it has two collection name keys